style.css

```diff
@@ -0,0 +1,918 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:wght@300;400;500;600;700;800&family=Inter:wght@100;300;400;500;600;700;800&display=swap');
+
+:root {
+  --primary-font: "Crete Round", sans-serif;
+  --secondary-font: "Inter", sans-serif;
+  --primary-color: #ff6b00;
+  --secondary-color: #f55800;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  border: 1 solid green;
+}
+
+body {
+  font-family: sans-serif;
+}
+
+.header-container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
```

Comment:
I have noticed you have repeated the display: flex property multiple times. Kindly it's good practice to use the DRY principle to help avoid repeating a property more than it should 😃 . An excellent example of a DRY principle that you can use on all repetitive containers.

`.flex-container {
  display: flex;
  flex-direction: row;
}`